doc/contribute/index.rst

```diff
@@ -761,3 +761,66 @@ Code component README template
 ==============================
 
 .. literalinclude:: code_component_README
+
+
+Contribution Roles and Responsibilities
+***************************************
+
+Zephyr community has a number of working groups and community members that
+work together in a development process flow to reviews PRs and Issue reports
+(Features/Enhancements/bugs). These PRs and Issue reports are used to help
+with the feature enhancements and quality improvements of Zephyr through
+regular releases as outlined in the Program Management wiki.
+
+To manage the volume of PRs and Issue reports within the project there are
+specific roles and responsibilities that are followed within the development
+process flow.
+
+PRs
+===
+
+**Zephyr Community**
+    • Provide timely review and disposition of submitted PRs that are not
+      marked as <DNM>.
```

Comment:
or Draft

Reply:
This entire paragraph (and review comment) are gone in the latest force push.

Reply:
yes, it was pointed out that there is related content in the other file I updated so I modified things (plus there was an objection for me explicitly calling out roles).